code/modules/client/preference_setup/loadout/items/general.dm

```diff
@@ -323,7 +323,7 @@
 	path = /obj/item/device/versebook/gadpathur
 	origin_restriction = list(/singleton/origin_item/origin/gadpathur)
 
-/datum/gear/aurora_miniature
+///datum/gear/aurora_miniature //NEW HORIZONS EDIT
```

Comment:
Shouldn't this whole thing be commented out?

Reply:
For the miniature? Can honestly be, just did the lazy "kill the source" move.
If its the Gadpathur verse book, can also be done but managed to ghetto a rephrase.

Reply:
For the miniature, yeah.

Reply:
Done